Source/FieldSolver/SpectralSolver/SpectralAlgorithms/GalileanAlgorithm.H

```diff
@@ -0,0 +1,33 @@
+#ifndef WARPX_GALILEAN_ALGORITHM_H_
+#define WARPX_GALILEAN_ALGORITHM_H_
+
+#include <SpectralBaseAlgorithm.H>
+
+/* \brief Class that updates the field in spectral space
+ * and stores the coefficients of the corresponding update equation.
+ */
+class GalileanAlgorithm : public SpectralBaseAlgorithm
+{
+    public:
+        GalileanAlgorithm(const SpectralKSpace& spectral_kspace,
```

Comment:
please add a Doxygen string that describes the parameters. [Here is a little example](https://github.com/openPMD/openPMD-api/blob/a545854367624984555a1d2a06463e88796df3fd/include/openPMD/auxiliary/Date.hpp#L30-L35).